packages/base-crypto/src/VirgilStreamSigner.ts

```diff
@@ -44,14 +44,21 @@ export class VirgilStreamSigner {
     validatePrivateKey(privateKey);
     const lowLevelPrivateKey = getLowLevelPrivateKey(privateKey);
 
-    const result = this.signer.sign(lowLevelPrivateKey);
+    let result: Uint8Array | undefined;
+    try {
+      result = this.signer.sign(lowLevelPrivateKey);
+    } catch (error) {
+      lowLevelPrivateKey.delete();
```

Comment:
since you won't need `lowLevelPrivateKey` anymore, you can replace `catch` with `finally` and remove `.delete()` call after this block